<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Imagenes/Fondo.jfif);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>87</y>
      <width>641</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Fall in love by Pucca</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Fall in love by Pucca</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SinglePlayer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>239</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1 Jugador</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MultiPlayer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>239</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2 Jugadores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scoreboard">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>239</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tabla de puntajes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>239</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>SinglePlayer</zorder>
   <zorder>MultiPlayer</zorder>
   <zorder>Scoreboard</zorder>
   <zorder>Exit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
